lib/muffin/base.rb

```diff
@@ -19,5 +21,15 @@ def initialize(request: nil, params: nil, scope: nil)
 
       assign_attributes
     end
+
+    def persisted?
+      attributes[:id].present?
+    end
+
+    # fields_for checks wether an object responds to [foo_attributes=]
+    def respond_to_missing?(method_name, include_private = false)
+      return true if method_name.to_s[/_attributes=\Z/]
+      super
+    end
```

Comment:
This is crucial for fields_for to work. If you are going to do `fields_for :foo` It checks, wether the object responds to `foo_attributes=`. So in order to be able to use fields_for, we have to respond to this and to most general way is something like this.